modules/light-clients/08-wasm/types/gas_register.go

```diff
@@ -1,21 +1,15 @@
 package types
 
 import (
-	"math"
-
-	wasmvm "github.com/CosmWasm/wasmvm"
 	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
 
 	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
 	storetypes "cosmossdk.io/store/types"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-
-	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// Copied subset of gas features from wasmd
-// https://github.com/CosmWasm/wasmd/blob/v0.31.0/x/wasm/keeper/gas_register.go
```

Comment:
should we leave this with updated link?

Reply:
also happy to just put this in a doc in the types directory if you want to keep the file line for line the same

Reply:
We have a link to the `main` branch in `gas_register_custom.go`, do you think that is enough?